src/trustyai/utils/tyrus.py

```diff
@@ -0,0 +1,368 @@
+            )
+            plots[output_name] = plot
+        return plots
+
+    # === DASHBOARD ================================================================================
+    def _get_plots(self):
+        """Grab all the plots and combine into one single Panel"""
+        cf_figures = self._get_byproduct_cf_plot()
+        lime_figures = self.lime_saliencies._get_bokeh_plot_dict()
+        shap_figures = self.shap_saliencies._get_bokeh_plot_dict()
+        output_names = list(cf_figures.keys())
+        tabs = []
+        for k in output_names:
+            title = str(k)
+            lime_tabbed = Tabs(
+                tabs=[
+                    Panel(child=lime_figures[k], title="LIME"),
+                    Panel(
+                        child=Div(text=LIME_TEXT.format(output_html(title))),
+                        title="About LIME",
+                    ),
+                ]
+            )
+            shap_tabbed = Tabs(
+                tabs=[
+                    Panel(child=shap_figures[k], title="SHAP"),
+                    Panel(
+                        child=Div(text=SHAP_TEXT.format(output_html(title))),
+                        title="About SHAP",
+                    ),
+                ]
+            )
+
+            cf_tabbed = Tabs(
+                tabs=[
+                    Panel(child=cf_figures[k], title="Available Counterfactuals"),
+                    Panel(
+                        child=Div(
+                            text=CF_TEXT.format(output_html(title)),
+                        ),
+                        title="About Counterfactuals",
+                    ),
+                ]
+            )
+
+            trustyai_content = GridBox(
+                children=[
+                    (lime_tabbed, 0, 0, 1, 1),
+                    (shap_tabbed, 1, 0, 1, 1),
+                    (cf_tabbed, 0, 1, 2, 2),
+                ]
+            )
+            joint = column(
+                Div(
+                    text="<h1>TrustyAI: Explaining {}</mark></h1>".format(
+                        output_html(title)
+                    )
+                ),
+                trustyai_content,
+                sizing_mode="scale_width",
+            )
+            tabs.append(Panel(child=joint, title=title))
+
+        full_dash = Tabs(tabs=tabs, sizing_mode="scale_width")
+        return full_dash
+
+    def run(self, display=True):
+        r"""Launch Tyrus TrustyAI Assistant and launch the dashboard. Depending on the setting
+        of ``notebook``, this will either automatically open the Tyrus visualizations
+        in a Jupyter notebook or browser window.
+
+         Parameters
+        ----------
+        display = True : boolean
+            Whether to automatically display the dashboard (true) or simply return it (false).
+        """
+        self._generate_saliencies()
+        self._generate_counterfactual_datasource()
+        plots = self._get_plots()
+
+        if display:
```

Comment:
```suggestion
        if display:
            return show(plots)

        return plots
```

To fix the linter error.

Reply:
I've gone for returning `None` otherwise

Reply:
which did not work, oops